dask_planner/src/expression.rs

```diff
@@ -314,6 +314,15 @@ impl PyExpr {
                 .iter()
                 .map(|arg| PyExpr::from(arg.clone(), self.input_plan.clone()))
                 .collect()),
+            Expr::InList { expr, list, .. } => {
+                let mut operands: Vec<PyExpr> = Vec::new();
+                operands.push(PyExpr::from(*expr.clone(), self.input_plan.clone()));
+                for list_elem in list {
+                    operands.push(PyExpr::from(list_elem.clone(), self.input_plan.clone()));
+                }
+
+                Ok(operands)
```

Comment:
Is there a more functional way to create a vector with `Expr` and all elements from `list`? 
cc: @andygrove

Reply:
@ayushdg This would be my suggestion:



```rust

                let mut exprs: Vec<Expr> = vec![expr.as_ref().clone()];
                exprs.extend_from_slice(list.as_slice());
                Ok(exprs
                    .iter()
                    .map(|e| PyExpr::from(e.clone(), self.input_plan.clone()))
                    .collect())
```